src/active-response/utilities.h

```diff
@@ -0,0 +1,77 @@
+#ifndef _UTILITIES_H_
```

Comment:
Don't create this file. We have already `src/active-response/active_responses.h` for common functions.